code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -20,13 +20,18 @@
 /obj/structure/closet/emcloset/anchored
 	anchored = TRUE
 
+/obj/structure/closet/emcloset/Initialize(mapload)
+	if (prob(1))
+		return INITIALIZE_HINT_QDEL
+	return ..()
```

Comment:
Do it before otherwise we run into issues with spawning stuff inside ourself and thats just weird